Automattic/wp-calypso
# Sidebar Upsell Nudge: Improve UI

Related to:
- https://github.com/Automattic/wp-calypso/issues/94476

## Proposed Changes

* Changed the UI according to the design supplied on the parent issue.

## Why are these changes being made?


* Redesigned the upsell nudge to improve the overall UX.

## Testing Instructions



* Open the Calypso live link.
* Use a site that hasn't registered a domain within the last 7 days.
* or alternatively hack the jitm engine on wpcom to show the the jitm
* You should see the jitm with the new design.

| Before | After |
| - | - |
![Screenshot 2024-12-10 at 17 43 31](https://github.com/user-attachments/assets/16d3e3dc-0460-448e-a3ab-3c9f589e3098)|![Screenshot 2024-12-10 at 17 43 13](https://github.com/user-attachments/assets/bf9ea0f0-22ec-4058-a9b1-56ce1abde9d7)

## Pre-merge Checklist
